src/Storages/tests/gtest_transform_query_for_external_database.cpp

```diff
@@ -91,14 +103,28 @@ struct State
                 context,
                 table_name,
                 std::make_shared<StorageMemory>(
-                    StorageID(db_name, table_name), ColumnsDescription{getColumns()}, ConstraintsDescription{}, String{}));
+                    StorageID(db_name, table_name), ColumnsDescription{tab.columns}, ConstraintsDescription{}, String{}));
         }
         DatabaseCatalog::instance().attachDatabase(database->getDatabaseName(), database);
+        // DatabaseCatalog::instance().attachDatabase("system", mockSystemDatabase());
+
         context->setCurrentDatabase("test");
     }
+
+    DatabasePtr mockSystemDatabase()
```

Comment:
This function seems unused.

Reply:
https://github.com/ClickHouse/ClickHouse/pull/48162